parser/composer.go

```diff
@@ -23,10 +29,64 @@ func ComposeBroadcastMessage(query message.Query) []byte {
 	return b
 }
 
-/*
- * Function that create the response to the sender
- * we suppose error is in the good range
- */
+// ComposeBroadcastStateLAO outputs a message confirming a LAO update after the signature threshold has been reached
+func ComposeBroadcastStateLAO(lao event.LAO, laoData message.DataCreateLAO, orgPublicKey string, lastSig []byte) (queryStr []byte, err_ error) {
+	//compose state update message
+	state := message.DataStateLAO{
+		Object:       "lao",
+		Action:       "state",
+		ID:           []byte(lao.ID),
+		Name:         lao.Name,
+		Creation:     lao.Creation,
+		LastModified: time.Now().Unix(),
+		Organizer:    []byte(lao.OrganizerPKey),
+		Witnesses:    laoData.Witnesses,
+	}
+
+	stateStr, errs := json.Marshal(state)
+	if errs != nil {
+		return nil, errs
+	}
+
+	content := message.Message{
+		Data:   stateStr,
+		Sender: []byte(orgPublicKey),
+		// signature from received Message
+		Signature:         lastSig,
```

Comment:
we compose the response so WE send the message (sender is orgPublicKey) and in the futur an other organizer would want to store the message and check the signature.
I've a doubt if it's not a mistake to put the last signature

Reply:
mhhh nope the last sig is wrong... Ideally we should compute the message, and ask the front end to sign it for us (as he's the one having the public key)...

Or we can generate a key pair for the backend... we shall check with the be2 to see what they did

Reply:
okay, fixed it